src/coreComponents/mainInterface/ProblemManager.cpp

```diff
@@ -848,7 +848,13 @@ map< std::tuple< string, string, string, string >, localIndex > ProblemManager::
 
                   localIndex const numQuadraturePoints = FE_TYPE::numQuadraturePoints;
 
-                  feDiscretization->calculateShapeFunctionGradients< SUBREGION_TYPE, FE_TYPE >( X, &subRegion, meshData, finiteElement );
+#if !defined(GEOSX_USE_CUDA)
+                  feDiscretization->calculateShapeFunctionGradients< SUBREGION_TYPE,
+                                                                     FE_TYPE >( X,
+                                                                                &subRegion,
+                                                                                meshData,
+                                                                                finiteElement );
```

Comment:
This seems to be causing some test failures on Lassen (`FAIL RUN`).

Reply:
I will include this optimization in another PR (https://github.com/GEOSX/GEOSX/pull/2123), where I can use the `formulation` tag to disable this calculation only for `SEM`formulations. I am therefore reverting this file.

Reply:
With this revert, all the integrated tests now pass on both Quartz and Lessen. I'm putting it back in the merge queue.